### PR TITLE
 KINNO-625-settings_entry

### PR DESCRIPTION
**Pull request by shayb**
#### Main purpose:
integrating send module to skd

#### Technical description:
1 update libraries to be align with the module libraries
2 add setting item that will trigger touch indicator

#### Dilemmas you faced with? 
need to check libraries changes and how it effects the clients that use it
#### Tests:
(Main tests you added/changed)
#### Documentation:
(Source/readme.md)

### CODE REVIEW

File: core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSource.kt [hunk 1]
```diff
@@ -6,6 +6,10 @@ interface SettingsDataSource {
 
     fun isBackedUp(publicAddress: String): Boolean
 
+    fun hasSeenTransfer(kinUserId: String): Boolean
```
Comment (OrenZak) [inline core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSource.kt:9]: add `Flow`postfix
can you change `isSaw` to `hasSeen` also ?

File: core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSourceImpl.kt [hunk 2]
```diff
@@ -1,6 +1,13 @@
 package com.kin.ecosystem.core.data.settings
 
 class SettingsDataSourceImpl(private val local: SettingsDataSource.Local) : SettingsDataSource {
+    override fun hasSeenTransfer(kinUserId: String): Boolean {
```
Comment (OrenZak) [inline core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSourceImpl.kt]: move it to the EOF

File: dependencies.gradle [hunk 3]
```diff
@@ -2,7 +2,8 @@ ext {
 
     //Version
     kinCoreVersion = '95645e1c17'
-    constraintLayoutVersion = '1.1.2'
+    constraintLayoutVersion = '1.1.3'
+
```
Comment (OrenZak) [inline dependencies.gradle]: remove

File: sdk/src/main/java/com/kin/ecosystem/history/view/KinDashView.java [hunk 4]
```diff
@@ -28,7 +28,7 @@ public KinDashView(Context context, AttributeSet attrs) {
             dashLength = styledAttributes.getDimensionPixelSize(R.styleable.KinEcosystemDashView_dashLength, 5);
             dashThickness = styledAttributes.getDimensionPixelSize(R.styleable.KinEcosystemDashView_dashThickness, 3);
             color = styledAttributes.getColor(R.styleable.KinEcosystemDashView_color, 0xff000000);
-            orientation = styledAttributes.getInt(R.styleable.KinEcosystemDashView_orientation, ORIENTATION_HORIZONTAL);
+            orientation = styledAttributes.getInt(R.styleable.KinEcosystemDashView_orientationValue, ORIENTATION_HORIZONTAL);
```
Comment (OrenZak) [inline sdk/src/main/java/com/kin/ecosystem/history/view/KinDashView.java:31]: sababa

File: core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSource.kt (hunk 1 above)
Comment (shayb) [reply core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSource.kt:9]: done

File: core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSourceImpl.kt (hunk 2 above)
Comment (shayb) [reply core/src/main/java/com/kin/ecosystem/core/data/settings/SettingsDataSourceImpl.kt]: done

File: dependencies.gradle (hunk 3 above)
Comment (shayb) [reply dependencies.gradle]: done

**Pull request merged**